static/schemas/2.0/skus.2.0.json

```diff
@@ -0,0 +1,248 @@
+{
+    "title": "Skus.2.0",
+    "definitions": {
+        "SkusRules": {
```

Comment:
I didn't know the history, maybe we intentionally wanted to pick a portion, @benbp, could you please confirm?